assets/javascripts/modules/validatorFocus.js

```diff
@@ -7,10 +7,26 @@
     }
 
     $('.error-summary a').on('click', function(e) {
+      var $this = $(this);
+      var focusId = $this.attr('data-focuses');
+      if (!focusId)
+      {
```

Comment:
brackets on same line is the general format around `AF` - we don't enforce this (yet) but IMO it would be nice to have code looking similar